build.gradle.kts

```diff
@@ -68,3 +69,23 @@ tasks.named<DependencyUpdatesTask>("dependencyUpdates").configure {
         isNonStable(candidate.version)
     }
 }
+
+forbiddenApis {
+    bundledSignatures = setOf("jdk-deprecated", "jdk-non-portable")
+    signaturesFiles = files("/config/forbidden-apis/forbidden-apis.txt")
+    ignoreFailures = false
+}
+
+subprojects {
```

Comment:
We do not use `subprojects` any more. Please move it to the `buildSrc`.
See `buildSrc/src/main/kotlin/pg-index-health.pitest.gradle.kts` for example